pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -166,8 +167,13 @@ func (s *Server) initDNSCerts(hostname, customHost, namespace string) error {
 			}
 		}
 	} else {
-		log.Infof("User specified cert provider: %v", features.PilotCertProvider.Get())
-		return nil
+		customCACertPath := security.DefaultRootCertFilePath
```

Comment:
As PilotCertProvider can be "kubernetes", "istiod", "custom" and "none", should there be a "else if features.PilotCertProvider.Get() == constants.CertProviderNone" section?

Reply:
The none case should've been ruled out by https://github.com/istio/istio/blob/master/pilot/pkg/bootstrap/server.go#L903-L905 and https://github.com/istio/istio/blob/master/pilot/pkg/bootstrap/server.go#L931-L933.